.github/workflows/build-be.yml

```
@@ -48,15 +48,19 @@ jobs:
 
   build-linux:
     runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-22.04
```

Why isn't the pull request just this?

---

Currently when built with `runs-on: ubuntu-latest` ubuntu-24.04 is used and makes GLIBC 2.38 the minimum requirement. Changing to `runs-on: ubuntu-22.04` does drop the GLIBC requirement down to 2.34. However I have concerns about how long will GitHub offer ubuntu-22.04 as a runtime. According to https://github.com/actions/runner-images, I expect them to drop ubuntu-22.04 when ubuntu-26.04 is available.

By adding the container directive, the build process is run inside of an ubuntu-20.04 container on the ubuntu-latest host. The availability of that container should never change, allowing better control over the build environemnt.

Originally when deciding to target ubuntu-20.04, I was hoping to get the GLIBC requirement down to around 2.29-2.31. The resulting libdoorstop.so has a requirement of just GLIBC 2.7.

While I believe either ubuntu-20.04 or ubuntu-22.04 would be good build targets. I don't know if locking the build system to GitHubs availability of only the two most recent Ubuntu LTS releases is option.